README.md

```diff
@@ -290,7 +290,7 @@ Results:
     score: 5,
     scoreText: '5',
     url: 'https://play.google.com/store/apps/details?id=com.dxco.pandavszombies&reviewId=Z3A6QU9xcFRPRWZaVHVZZ081NlNsRW9TV0hJeklGSTBvYTBTUlFQUUJIZThBSGJDX2s1Y1o0ZXRCbUtLZmgzTE1PMUttRmpRSS1YcFgxRmx1ZXNtVzlVS0Zz'
-    title: 'I LOVE IT',
+    title: 'I LOVE IT', //GOOGLE DO NOT RETURN THE TITLE ANYMORE
```

Comment:
then remove it from the documentation

Reply:
I'm still returning this as ```null```, otherwise it will be another breaking change.
Should I remove or just change the documentation?